app/validators/profile_validator.rb

```diff
@@ -32,19 +32,10 @@ def validate(record)
   private
 
   def summary_too_long?(record)
-    # Calling the summary attribute method on record during onboarding
-    # throws a NoMethodError
-    return unless record.respond_to?(SUMMARY_ATTRIBUTE)
```

Comment:
This check should not longer be needed. Since `summary` is an actual column on `profiles` now records will always respond to it.